pkg/host-timesync-daemon/main.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/linuxkit/virtsock/pkg/vsock"
```

Comment:
I tried moving it to the end, then `gofmt -w -s` puts it back! I was confused by this so I did a little more experimentation. It seems that inserting a blank line between the block of regular imports and the github.com ones has the desired effect. I think it must be alphabetical within each newline-delimited block. I'll modify the patch.